tests/integration/api/conftest.py

```diff
@@ -93,17 +89,36 @@ def test_metadata_entity() -> (
     return file
 
 
+@pytest.fixture
+def test_suspicious_metadata_entity() -> dict[str, Any]:
+    """Represent a test metadata object."""
+    file: dict[str, Any] = DocMetadataEntity(
```

Comment:
Are you guys only testing doc metadata right now? If so that's not an issue as the schema changes should make it so you don't have to declare the type and can test whatever you want.